public/app/features/explore/Logs.tsx

```diff
@@ -166,6 +169,17 @@ export class UnthemedLogs extends PureComponent<Props, State> {
     }
   };
 
+  onChangePrettifyLogMessage = (event?: React.SyntheticEvent) => {
+    const target = event && (event.target as HTMLInputElement);
```

Comment:
If you're getting type errors from using `event.target`, try `even.currentTarget`, so you don't have to type cast. Also, is there a reason why the `event` object is optional?

Reply:
Oh i don't know actually, i looked at the other "onChange" functions and saw that they had the event as optional so i did the same. Thanks for the tip regarding `event.currentTarget`, will use that instead 👍

Reply:
It seems i'm getting an error when using `event.currentTarget`. Error:`Property 'checked' does not exist on type 'EventTarget & Element'.`. Maybe that's why i need to typecast with `event.target`?

Reply:
You might need to play around a bit with the typing. Try `React.ChangeEvent<HTMLInputElement>` for the event type instead.